UnityProject/Assets/Scripts/Objects/Other/PineTreeXmas.cs

```diff
@@ -19,47 +19,55 @@ public class PineTreeXmas : NetworkBehaviour, ICheckedInteractable<HandApply>
 		[SerializeField] private GameObject giftObject;
 		[SerializeField] private AddressableAudioSource ambientReminder;
 
-		[SyncVar] private List<string> giftedPlayers = new List<string>();
+		private List<string> giftedPlayers = new List<string>();
 		private bool canPickUpGifts;
 
 		private SpriteHandler spriteHandler;
 
-		private void Awake()
+		private void Start()
 		{
 			spriteHandler = GetComponentInChildren<SpriteHandler>();
 			if (TimedEventsManager.Instance.ActiveEvents.Contains(eventData))
 			{
 				canPickUpGifts = true;
 				spriteHandler.SetSpriteSO(xmasSpriteSO);
-				PlayEasterEgg();
+				UpdateManager.Add(PlayEasterEgg, 60);
```

Comment:
Remember to remove the update in OnDisable()

Reply:
how do you remove `UpdateManager.Add(PlayEasterEgg, 60);`?
Apparently you can only remove updates that use Callbacks

Reply:
`UpdateManager.Remove(CallbackType.PERIODIC_UPDATE, PlayEasterEgg);`

Reply:
thats a bit confusing, anyways; done.